ibm/mas_devops/roles/ibm_catalogs/defaults/main.yml

```diff
@@ -15,4 +15,4 @@ mas_catalog_digests:
   v8-220927-amd64: sha256:37d492a851137567bcfea0b5530fca05374a08a78b41afb712e17c2542016ae7
   v8-221025-amd64: sha256:993d5ca2f327494abf3dc7175968bc12165a341c6f1f95a0d50ecde6a5663c7f
   v8-221129-amd64: sha256:56905868dcf7ae8ca0901cfd92225a3f52db250c091987fe81b149dbacb4df13
-  v8-221228-amd64: sha256:505a135ba1af2cee548703e85b17764074b7479dbbf506b132d5871af793073f
+  v8-230111-amd64: sha256:445f39b3628f3ff7b78a775349321a712175963ac41915bd0cb7985149bea752
```

Comment:
Reminder: update this digest once we have the real one after releasing the catalog to ICR